django-stubs/db/models/query.pyi

```diff
@@ -86,7 +86,7 @@ class QuerySet(Iterable[_T], Sized):
     def last(self) -> Optional[_T]: ...
     def in_bulk(
         self, id_list: Any = ..., *, field_name: str = ..., **kwargs: Any
```

Comment:
Could you replace `id_list` parameter annotation with the `Sequence[Any]`, and `field_name` with `Optional[str]` while you're here?

Reply:
Hi. I've changed the id_list annotation. I also made a few other improvements and added a test.